Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -235,26 +235,27 @@ func readDocuments(matching query: Query) {
 
 func listenToDocument(at docRef: DocumentReference) {
 
-    let listener = docRef.addSnapshotListener() { document, error in
-        if let error = error {
-            print("Uh oh! Listen canceled: \(error)")
-            return
-        }
+  let listener = docRef.addSnapshotListener() { document, error in
+    if let error = error {
+      print("Uh oh! Listen canceled: \(error)")
+      return
+    }
 
-        if let document = document {
-            // Note that document.data() is nullable.
-            let data : [String:Any]? = document.data()
-            print("Current document: \(data)");
-            if (document.metadata.isFromCache) {
-                print("From Cache")
-            } else {
-                print("From Server")
-            }
-        }
+    if let document = document {
+      // Note that document.data() is nullable.
+      if let data : [String:Any] = document.data() {
+        print("Current document: \(data)");
+      }
+      if (document.metadata.isFromCache) {
```

Comment:
Swift doesn't require parens in if statements 😛

Reply:
In my defense that was there before I merely reformatted this statement :-)